pkg/cli/advisory_export.go

```diff
@@ -86,12 +97,27 @@ type exportParams struct {
 	advisoriesRepoDirs []string
 
 	outputLocation string
+
+	// format controls how commands will produce their output.
+	format string
 }
 
+const (
+	// OutputDefault default output.
+	OutputDefault = ""
+	// OutputJSON JSON output.
+	OutputYAML = "yaml"
+	// OutputCSV CSV output.
+	OutputCSV = "csv"
+)
+
 func (p *exportParams) addFlagsTo(cmd *cobra.Command) {
 	addNoDistroDetectionFlag(&p.doNotDetectDistro, cmd)
 
 	cmd.Flags().StringSliceVarP(&p.advisoriesRepoDirs, "advisories-repo-dir", "a", nil, "directory containing an advisories repository")
 
 	cmd.Flags().StringVarP(&p.outputLocation, "output", "o", "", "output location (default: stdout)")
+
+	cmd.Flags().StringVarP(&p.format, "format", "f", "",
+		fmt.Sprintf("Output format. One of: [%q, %q, %q]", OutputDefault, OutputYAML, OutputCSV))
```

Comment:
nit: I might use Cobra's mechanism for setting the default value, rather than managing it ourselves. This also means we won't suggest `""` as a value for `--format` in the help text, which I think is a nice (but super tiny) improvement in clarity. So something like this:

```suggestion
	cmd.Flags().StringVarP(&p.format, "format", "f", OutputCSV,
		fmt.Sprintf("Output format. One of: [%s]", strings.Join([]string{OutputYAML, OutputCSV}, ", ")))
```

Also, if you cascade this change to lines 59 and 62 as well, you end up not needing the `OutputDefault` value at all

Reply:
Cleaned this up - great suggestion!